docs/data/material/components/skeleton/Variants.js

```diff
@@ -8,6 +8,7 @@ export default function Variants() {
       <Skeleton variant="text" />
       <Skeleton variant="circular" width={40} height={40} />
       <Skeleton variant="rectangular" width={210} height={118} />
+      <Skeleton variant="rounded" width={210} height={118} />
```

Comment:
For a reason I ignore, this does not update the documentation render on my local:

<details>
![image](https://user-images.githubusercontent.com/1698357/172163292-20cdb946-02ee-4e49-9726-ce569448b575.png)
</details>

I would like help on that part! :-)

Reply:
Found it, I missed the related `Variants.tsx` part.